ui/components/Navigator.js

```diff
@@ -301,6 +301,14 @@ const categories = [
         show: false,
         link: true,
       },
+      {
+        id: "Filters",
+        icon: <img src="/static/img/pattern_trans.svg" style={{ width: "1.21rem" }} />,
```

Comment:
Eventually, we will have a new icon for filters. Thanks for reusing an existing one in the meantime.

Reply:
Yes, I will be updating it once it's working.